### DISCUSSION TITLE
Qubino Mini Dimmer S2 security stops responding

### DISCUSSION THREAD

**Discussion by thecobra666** [zwave-js/zwave-js#4438] [Q&A] [answered]
I'm not opening a bug report for this since I think it's a firmware issue instead of a zwavejs issue but I would like to hear if there is anyone else who has experienced this and maybe already opened a ticket with qubino.

I have 3 of these Qubini Mini Dimmers installed. 2 are on FW 20.2 and 1 is on FW 20.0. All 3 of them stop responding after 2 to 4 days and a powercycle is needed to bring them back.

When joining them with S0 or unauthenticated I did not had this issue. I've only found 1 topic which mentiones this same problem but, it's from 2020 and no further feedback was posted. https://forum.z-wave.me/viewtopic.php?t=33681

So a shoutout if anyone else has noticed this behaviour and if yes, did you find a solution or did you get a solution from Qubino for this?

In the mean time I have opened a ticket with Qubino and will post feedback here.

**Comment by Thaui57** [comment]
I do have a similar problem with quibino flush shutter. Response from the quibino support team: include the unit with no security but this doesn't help as well. The device is running for 2-4 day and suddenly stops reporting values.

**Reply by thecobra666** [reply]
> I do have a similar problem with quibino flush shutter. Response from the quibino support team: include the unit with no security but this doesn't help as well. The device is running for 2-4 day and suddenly stops reporting values.  

They have answered that they are going to test this and report back to me in a few days. Hopefully they have the same issue and are willing to fix it.

**Comment by Thaui57** [comment]
I got the same reply. Answer was they have no issues foundAm 02.04.2022 09:54 schrieb thecobra666 ***@***.***>:

I do have a similar problem with quibino flush shutter. Response from the quibino support team: include the unit with no security but this doesn't help as well. The device is running for 2-4 day and suddenly stops reporting values.

They have answered that they are going to test this and report back to me in a few days. Hopefully they get have the same issue and are willing to fix it.

**Reply by thecobra666** [reply]
Great, that means getting them out of the wall and onto no security anymore. I'll wait a little longer but damm, these qubino devices have bugs.

**Comment by thecobra666** [comment, marked as answer]
So they responded back with me that this is indeed a bug in the firmware. They do not do OTA updates so I would need to send both devices back to them to get this fixed (which I will not do). 

TLDR: only join them without security..

**Reply by AlCalzone** [reply]
We should mention this as a known issue in the configuration DB. Can you tell me which firmware ranges are affected?

**Reply by thecobra666** [reply]
> We should mention this as a known issue in the configuration DB. Can you tell me which firmware ranges are affected?

I had the issue with FW 20.2 and FW 20.0.

**Reply by AlCalzone** [reply]
Any chance you know which is the version where this is fixed? We have a handful of users on v21.6, this might be it.

**Reply by thecobra666** [reply]
> Any chance you know which is the version where this is fixed? We have a handful of users on v21.6, this might be it.

I will ask them :)

**Reply by thecobra666** [reply]
@AlCalzone This is the respons I got from Qubino: Version of Mini Dimmer is 31.00 (This version has SmartStart implemented, autoinclusion is not available anymore), problem occured on version 2.02.

**Reply by AlCalzone** [reply]
> problem occured on version 2.02.

You mean 20.2? Didn't you say you're having the issue on 20.0 too?

**Reply by thecobra666** [reply]
> > problem occured on version 2.02.
> 
> You mean 20.2? Didn't you say you're having the issue on 20.0 too?

Hi, yeah 20.2 :) sorry early morning :D I did had the issue on 20.0 also. Since I only had these 2 versions to test with, I can't comment on the rest. Only that Qubino said this should be resolved with version 31.00. My dimmers are on 20.2 and 20.0.

**Reply by AlCalzone** [reply]
https://github.com/zwave-js/node-zwave-js/pull/4515 will add this info to the device file (and later the config DB)

**Reply by andras-tim** [reply]
I have only one **Qubino Mini Dimmer**, but mine is working properly with the `v20.0` firmware + S2 since I replaced my **Aeotec Z-Stick 7** stick with **Aeotec Z-Stick Gen5+** _(4 months ago)_

More info:

* **[13-134-0-libraryType] Library type:** [3] Enhanced Slave
* **[13-134-0-protocolVersion] Z-Wave protocol version**: 5.3
* **[13-134-0-firmwareVersions] Z-Wave chip firmware versions**: ["20.0","20.0"]
* **[13-134-0-hardwareVersion] Z-Wave chip hardware version**: 2

**Comment by Thaui57** [comment]
No idea sorry.
